app/views/diaper_drive_participants/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "#{current_organization.name} - Agencies - Diaper Drive Participants - #{@diaper_drive_participant.name}" %>
+
+<h1><%= @diaper_drive_participant.name %></h1>
+
+<ul>
+<% if @diaper_drive_participant.phone.present? %>
+  <li><%= @diaper_drive_participant.phone %></li>
```

Comment:
I didn't want to show `<li>...</li>` unless the field was present (ie. not showing an empty list bullet). Is there a better way to do this?